drivers/QSPI.h

```diff
@@ -28,6 +28,8 @@
 
 #define ONE_MHZ     1000000
 
+#define QSPI_NO_INST 0x00
```

Comment:
One additional request, please send new PR: Is this documented anywhere? There are in doxygen notes about: "use this value for this phase to be ignored" . This would be good to have  there as well otherwise this define is not known to a user.

Reply:
Submitted #11691